sdcm/cluster.py

```diff
@@ -5764,7 +5751,7 @@ def update_scrape_configs(base_scrape_configs, static_config_list, job_name="nod
                     # NOTE: monitoring-4.7 expects that node export metrics are part of exactly the "node_export" job
                     if scrape_config.get("job_name", "unknown") != job_name:
                         continue
-                    if "static_configs" not in base_scrape_configs[i]:
+                    if "static_configs" not in scrape_config:
```

Comment:
Not sure how this change connected with refactoring

Reply:
This changes was made by ruff. it looks relevant

Reply:
it's not relevent to this PR,

I would guess you are using a newer version of ruff, and not the one stated in `requirements.txt`

please remove this change

Reply:
Removed